aim/db/infra_model.py

```diff
@@ -131,9 +131,11 @@ class HostDomainMapping(model_base.Base, model_base.AttributeMixin):
     __tablename__ = 'aim_host_domain_mapping'
 
     host_name = sa.Column(sa.String(128), primary_key=True)
-
-    vmm_domain_name = sa.Column(sa.String(64))
-    physical_domain_name = sa.Column(sa.String(64))
+    domain_name = sa.Column(sa.String(64), primary_key=True, nullable=True)
+    domain_type = sa.Column(sa.Enum('PhysDom',
```

Comment:
We also need to make domain_type a primary key also right? Otherwise how do you handle the case that a host belongs to 2 domains with the same name but of different type (say physdom and openStack) ?

Reply:
Yes, you are correct. Will fix.